packages/ai/src/ai/account/store.py

```
@@ -164,6 +165,150 @@ async def delete_file(self, filename: str, expected_version: Optional[str] = Non
+
+        Returns:
+            Number of bytes written.
+        """
+        context['buffer'].extend(data)
+        return len(data)
+
+    async def close_write(self, filename: str, context: Any) -> None:
+        """Finalize and commit the write session."""
+        await self.write_bytes(filename, bytes(context['buffer']))
+
+    async def abort_write(self, filename: str, context: Any) -> None:
```

Ruff (0.15.7)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `context`

(ANN401)